### PR TITLE
Convert static site to data driven

### PR DESCRIPTION
**Pull request by glasnt**
TLDR: sorting is annoying, let's use data!

* Uses Jekyll Collection to drive conference and regional listings
* Dynamically sorts contents
* Leverages github's native jekyll support (via gh-pages gem)
* Reconfigured listing to scroll *horizontally* for listings
  * National conferences are sorted by LOCATION
  * Regional conferences are sorted by NAME
  * included a piece of getskeleton for nicer column listings, with removes having to add a 'half-col' split
* Updated README to suit

Sample of new website in [glasnt/pycon.org#1](https://github.com/glasnt/pycon.org/pull/1),  see specifically the [netlify preview](https://deploy-preview-1--vibrant-kirch-79fa36.netlify.com/) (highly recommend incorporating netlify PR testing. After integrated, merging a PR would use native gh-pages. [see more](https://opensource.com/article/19/8/github-pages-cicd-pipeline))

Code review @lorenanicole, infra review @ewdurbin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,75 +5,60 @@ This is the source code for [pycon.org](http://www.pycon.org/).
...
-    </div>
-  </div>
-  <div class="conference-details">
-    <p>PyConFR is the annual conference for the Francophone world, organized by the <a href="https://www.afpy.org/">AFPy</a>.</p>
-    <p>Past conferences have been held in Pau(2015), AnotherLocation(2014), AnotherLocation(2013).</p>
-    <p>The next PyConFR will be held in Rennes on the 13th-16th of October, 2016.</p>
-    <p><a href="https://2016.pycon.fr/">Visit Website</a></p>
-  </div>
-</div>
+---
```
Comment (lorenanicole) [inline README.md:30]: YES! I haven't actually used Jekyll ... Pelican is as far as I have gotten with static site generators 🎊

**Review by lorenanicole — APPROVED** [review]
LGTM @glasnt thanks for adding this in!

**Comment by jefftriplett** [comment]
LGTM to me. I'd suggest bumping Pycon Latam to the top list of PyCons (since it sticks out to me as the only PyCon named conferences in that section). I'd also like to add  DjangoCon US to the bottom section once this lands.

**Review by ewdurbin — APPROVED** [review]
Thank you for this excellent work @glasnt!

I don't have any blockers to merging, @jefftriplett's suggestion could be included in this PR or resolved after.

As far as netlify goes, I suppose we should merge this then configure it?

**Comment by glasnt** [comment]
@ewdurbin yup, happy to have this merged, so the Pycon Latam update seperately, setup netlify seperately (requires an repo admin to setup)
